ionic-angular-template/src/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm, FormGroup, FormBuilder, Validators, FormControl } from '@angular/forms';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit {
+  public signInForm: FormGroup;
+  public email: FormControl;
+  public password: FormControl;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private alertCtrl: AlertController,
+    private fb: FormBuilder
+  ) { }
+
+  ngOnInit() {
+    this.email = new FormControl('', Validators.compose([Validators.required, Validators.email]));
```

Comment:
These are already part of the form group. There is no need to add them separately. You can move the initial value, as well as the Validators in the creation of the form group.